src/commands.ts

```diff
@@ -317,7 +309,36 @@ export class CosmosDBCommands {
         }
         return null;
     }
-
+    public static async deleteDocDBDatabase(db: DocDBDatabaseNode, explorer: CosmosDBExplorer): Promise<void> {
+        const confirmed = await vscode.window.showWarningMessage("Are you sure you want to delete database '" + db.label + "' and its collections?",
```

Comment:
The user will get a null ref if they try to run this command from the command palette. Wrap this in 'if (db)' for now

We should probably leverage [this](https://code.visualstudio.com/docs/extensionAPI/extension-points#_context-specific-visibility-of-command-palette-menu-items) to hide commands from the palette until they work, but that should be a separate PR to fix https://github.com/Microsoft/vscode-cosmosdb/issues/35